source/client/index.js

```diff
@@ -26,6 +26,7 @@ module.exports = ({
   routes = ensureRoutes('createClientApp'),
   basepath = '',
   history = createHistory(),
+  shouldUpdateScroll = defaultUpdateScroll,
```

Comment:
Ah, fair enough. I was just using the function name as it's called in `scroll-behaviour` but I'll change it to be more explicit.